Library/Homebrew/shims/super/cc

```diff
@@ -134,7 +134,7 @@ class Cmd
     when :cxx
       cxxflags + args + cppflags
     when :ccE
-      args + cppflags
+      cflags + args + cppflags
```

Comment:
This feels like too big a change to be making if it applies only for ARM.

Reply:
It's only needed for `glibc` on ARM. I am happy to find a fix in the formula that works, but I couldn't. Any suggestions?

Side note: I am super new to Homebrew and it's formulas.

Reply:
@asutoshpalai I'd consider opening a discussion in Homebrew/discussions to brainstorm on the best fix. We don't support ARM on Linux so don't want to make changes like this for a single formula on an unsupported platform.

Reply:
Discussion on Glibc failure on aarch64: https://github.com/Homebrew/discussions/discussions/3881